lib/next_ls/runtime/sidecar.ex

```diff
@@ -31,4 +54,60 @@ defmodule NextLS.Runtime.Sidecar do
 
     {:noreply, state}
   end
+
+  defp filter_attributes(symbols) do
+    symbols
+    |> Enum.filter(&match?({:attribute, _, _, _}, &1))
+    |> Enum.reject(fn {_, "@" <> name, _, _} ->
+      Map.has_key?(Module.reserved_attributes(), String.to_atom(name))
+    end)
+  end
+
+  defp parse_symbols(file, module) do
+    ast = file |> File.read!() |> Code.string_to_quoted!(columns: true)
+
+    {_ast, %{symbols: symbols}} =
+      Macro.traverse(ast, %{modules: [], symbols: []}, &prewalk/2, &postwalk(&1, &2, module))
+
+    symbols
+  end
+
+  # add module name to modules stack on enter
```

Comment:
not gonna lie, i was not really sure as to the difference bewteen pre and post walk until i read this PR 😅